packages/main/src/plugin/onboarding/onboarding-init.ts

```
@@ -38,9 +51,35 @@ export class OnboardingInit {
+        await this.configurationRegistry.updateConfigurationValue(
+          `${WelcomeSettings.SectionName}.${WelcomeSettings.Version}`,
+          undefined,
+          CONFIGURATION_DEFAULT_SCOPE,
+        );
+        this.apiSender.send('onboarding:restart');
+      }),
+    );
+  }
+
+  dispose(): void {
+    this.#disposables.forEach(d => d.dispose());
```

Biome (2.4.14)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)